docs/abtest.rst

```diff
@@ -8,6 +8,39 @@
 A/B Testing
 ===========
 
+Nimbus experiments
+------------------
+
+Nimbus (Experimenter), is Mozilla's 1st-party experimentation platform.
+Experiments from this platform will split traffic before users arrive on
+the page (meaning no client-side JS delay).
+
+.. Note::
+
+    Bedrock code will still determine what templates to show based on locales
+    and the url parameters added by Nimbus.
+
+There are generally 3 user groups to consider:
+
+#. In experiment - control branch
+#. In experiment - treatment branch
+#. Not in experiment
+
+The Firefox team will set up the experiment and provide the URL params:
+
+- **branch** (required): identifies an experiment. When there is a single experiment, the branch value can be used to distinguish control and treatment (i.e. ``experiment-wnp-136-control`` and ``experiment-wnp-136-vpn``)
+
+- **variant** (optional): when multiple experiments run at the same time, this secondary param distinguishes control and treatment variants within each experiment (i.e. ``experiment-wnp-136-vpn`` has 2 variants and ``experiment-wnp-136-tabs`` has 4 variants). Nimbus variant must match pre-defined value (i.e. ``v1``, ``v2``, ``v3``, ``v4``).
+
+Bedrock automatically sets ``nimbus_branch`` and ``nimbus_variant`` variables
+for use in ``firefox/views.py`` logic.
+
+Even if `control branch` and `not in experiment` users are served the same
+template, it may be important to distinguish them from an analytics
+perspective. UTM values should be included in the Nimbus experiment brief.
+These values can be used on internal `analytics events <http://bedrock.readthedocs.io/en/latest/coding.html#variation-views>`_
+and on external URLs.
+
```

Comment:
note on timing? demo with experiment logic must be available by certain time before launch?